jetty-core/jetty-tests/jetty-test-client-transports/src/test/java/org/eclipse/jetty/test/client/transport/ServerRetainContentTest.java

```diff
@@ -52,7 +52,6 @@ protected void prepareServer(Transport transport, Handler handler) throws Except
     public void testRetainPOST(Transport transport) throws Exception
```

Comment:
No need to override `prepareServer()` above to just call super.

Use `@MethodSource("transportsNoFCGI")`

Remove the `assumeTrue()` below.